nephos/helpers/k8s.py

```diff
@@ -246,6 +246,25 @@ def secret_read(name, namespace="default"):
     return secret.data
 
 
+def secret_from_files(secret, namespace, keys_files_path):
```

Comment:
Can we not reuse much functionality from below function rather than copy-pasting most of the code?

Reply:
This comment still stands, we end up with two functions that are almost identical. I'd rather keep yours, @inzamam-iqbal, and remove the `secret_from_file` in favour of `secrets_from_files`